base/utils/tests/testthat/test-read.output.R

```diff
@@ -8,13 +8,17 @@ teardown(unlink(testdir, recursive = TRUE))
 test_that("returns a list or dataframe as requested", {
 	times_to_netcdf(0:364, "days since 2001-01-01", testdir, "2001.nc")
 	
-	listres <- read.output(runid = "", outdir = testdir, variables = "Y", dataframe = FALSE)
+        out_log <- capture.output(type = "message", {
+	  listres <- read.output(runid = "", outdir = testdir, variables = "Y", dataframe = FALSE)
+        })
```

Comment:
I see that you test for expected logger output below, but in these cases that expect success `out_log` goes unexamined. Would adding `expect_equal(out_log, "")` to each case that's supposed to succeed be sensible or paranoid?

Reply:
All of these cases are successful, but extremely chatty in terms of `PEcAn.logger`, which pollutes the `testthat` output and therefore the Travis log. So testing for silence would fail.

At some point in the future, it would be great to implement `expect_log` to check not just for message content but also for messages at/above a specific logger level, so we can be fine with tests that spit out info and debug messages, say, but not warnings and errors. But as I said above, I'd prefer to punt on that till a later PR.

Reply:
You should be able to use `logger.setLevel('WARN')` to only show the log messages at WARN or above.

Reply:
Ah yes, the chattiness. Suppressing that is good, I'm just thinking about what happens if something changes and makes these tests produce new warnings. If you're happy declaring test success in the case where it still passes the class/dimension/etc checks but complains more than it used to--which seems pretty reasonable to me--then this is good as it is.